eip-halvening.md

```diff
@@ -0,0 +1,68 @@
+---
+eip: <to be assigned>
+title: Update Network Block Rewards to Adhere to "Minimum Viable Issuance" Policy
+author: Kevin Owocki <kevin@gitcoin.co>
+discussions-to: Pull Request
+status: Draft
+type: Economic
```

Comment:
`Economic` isn't a valid type.  See for https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md and https://eips.ethereum.org/EIPS/eip-1 for valid types.